### ISSUE TITLE
[Suggestion] special monika

### ISSUE THREAD

**Issue by Rakoor** [Monika-After-Story/MonikaModDev#6003]
So, I've just finished totono, minor spoilers ahead:

You know in the end of the game. You have to answer 30 questions, their answer are completely randomly generated.

The questions are about the personality of the heroine. What she likes or hate or want ..etc.
30 question with 3 choices each. Makes around 205 trillion different personality. Which is amazing

So, i was thinking if that was possible with monika, it would give every monika a speciality, and feel unique, and it won't interfere with the "canonicity" of it because it can be small random things

I believe this is very easy to make. And would make a great feature... hopefully

**Comment by ThePotatoGuy** [comment]
Do you mean that Monika asks the player these questions? We sort of do that already when players answer questions that Monika asks - it's just not all at once.

**Comment by Rakoor** [comment]
> Do you mean that Monika asks the player these questions? We sort of do that already when players answer questions that Monika asks - it's just not all at once.

No, not necessarily asking the player, but rather Monika having slightly different personalities.

Like "My monika" likes Tennis, the color green, wanted to be a magic girl when she was young...etc. while "another Monika" might hate these things. And love other things...etc


So each monika for everyplayer would be, more personal.
And the more differences there are, the more combinations of them. It would be impossible for two monikas to be the same.

**Comment by ThePotatoGuy** [comment]
>Monika having slightly different personalities.

We kind of do this through branching her dialogue based on your answers to questions, but I doubt having each Monika be randomly different would fly with everyone.

converting to issue discussion.
